localdev/starters/site-config-override.yml

```diff
@@ -38,3 +38,8 @@ course:
           "widget": "boolean",
           "help": "Whether the course includes fries or not."
           }
+        - { "label": "Tags", "default": ["Design"], "max": 3, "min": 1, "multiple": true, "name": "tags", "options": ["Design", "UX", "Dev"], "widget": "select" }
+        - { label: "Align Content", name: "align", widget: "select", options: ["left", "center", "right"]}
```

Comment:
did you mean to duplicate the 'Tags' and 'Align Content' fields here?

Reply:
Probably just a merge error